### PR TITLE
[#490] Fix some issues and add more samples for iOS with SwiftUI

### LINKED ISSUE #490: Add more SwftUI samples
After the first merge of SwftUI samples (https://github.com/Orange-OpenSource/a11y-guidelines/pull/489), other sections must be looked and updated if needed after a big review of iOS-related pages.

# Topics to update or check if needed

- [ ] Truncation hyphen
- [ ] Graphical elements size
- [ ] Large Content Viewer
- [ ] Continuous adjustable values
- [ ] Custom actions
- [ ] Custom rotor
- [ ] Accessibility options
- [ ] Navigation bar
- [ ] Speech synthesis
- [ ] Switch Control
- [ ] Vocalized application name

### PR DESCRIPTION
**Pull request by pylapp**
The job is not completed about SwiftUI sections.
Some typography oad rendering issues remain, and several sections are empty.
This pull request brings more content for SwiftUI.

Closes #490

### CODE REVIEW

**Comment by pylapp** [comment]
@pya35 This pull request is still in draft ; I will notify here when it will be ready for review, I hope this week.

**Comment by pylapp** [comment]
@pya35 I completed my job ; now you can make a review and merge it if you agree :)

**Comment by pylapp** [comment]
@pya35 Did you get the time to review this contribution?

File: src/fr/mobile/ios/developpement.md [hunk 1]
```diff
@@ -2497,6 +2563,92 @@ class TruncationHyphen: UIViewController {
...
+        let localeRef = locale as CFLocale
+        guard CFStringIsHyphenationAvailableForLocale(localeRef) else {
+            return self
+        }
+        
+        let mutableSelf = NSMutableString(string: self)
+        var hyphenationLocations = Array<Bool>(repeating: false, count: count)
+        let range = CFRangeMake(0, count)
+        
+        for i in 0..<count {
```
Comment (pya35) [inline src/fr/mobile/ios/developpement.md:2627]: both "for" loops below break the page display, if removed it works

**Review by pya35 — APPROVED** [review]
OK

**Pull request merged**